src/components/video/examples/video/index.njk

```diff
@@ -0,0 +1,6 @@
+{% from "components/video/_macro.njk" import onsVideo %}
+{{ 
+    onsVideo({
+        "youtubeUrl": 'https://youtu.be/BiZk2pnSM6w'
```

Comment:
Youtube short URLs are blocked by most adBlockers, so it's best to use full URLs
```suggestion
        "youtubeUrl": "https://www.youtube.com/watch?v=BiZk2pnSM6w"
```